### PR TITLE
Change max response size to target response size

### PR DESCRIPTION
**Pull request by joshua-kim**
## Why this should be merged

Prevent edge-cases where a gossiped item might never get gossiped

## How this works

Changes the max gossip size to a target size

## How this was tested

Changed unit test

### CODE REVIEW

File: network/p2p/gossip/handler.go [hunk 1]
```diff
@@ -70,14 +70,15 @@ func (h Handler[T]) AppRequest(_ context.Context, _ ids.NodeID, _ time.Time, req
 			return false
 		}
 
-		// check that this doesn't exceed our maximum configured response size
+		// check that this doesn't exceed our maximum configured target response
+		// size
```
Comment (anusha-ctrl) [inline network/p2p/gossip/handler.go:74]: fix this comment?

**Review by patrick-ogrady — APPROVED** [review]

**Review by StephenButtolph — APPROVED** [review]
